products/sle/main.pm

```diff
@@ -1116,6 +1116,10 @@ elsif (get_var("WICKED")) {
     boot_hdd_image();
     load_wicked_tests();
 }
+elsif (get_var("NFV")) {
+    boot_hdd_image();
+    load_nfv_tests();
+}
```

Comment:
Better prevent the duplication by defining a complete "NFV" section including the boot_hdd_image part within lib/main_common.pm. We want products/{opensuse,sle}/main.pm to be harmonized and simplified further. IMHO the way to go is to put bigger common blocks together in lib/main_common.pm . So it could look like this:

```perl
## lib/main_common.pm
sub load_nfv_tests {
    boot_hdd_image;
    loadtest 'console/openvswitch';
}

## products/{opensuse,sle}/main.pm
load_nfv_tests if get_var('NFV');
```

One could go even further by also moving the last line together with more common lines in a helper function in lib/main_common.pm.

and please use single ticks by default unless you need variable expansion within strings.